src/rules/maxLineLengthRule.ts

```diff
@@ -19,6 +19,8 @@ import * as ts from "typescript";
 
 import * as Lint from "../lint";
 
+import { IDisabledInterval } from "../language/rule/rule";
+
 export class Rule extends Lint.Rules.AbstractRule {
```

Comment:
Can we declare the rule options as constants, and then use those constants in the rule metadata and in the code? 

(For example, [similar to this](https://github.com/palantir/tslint/blob/master/src/rules/tripleEqualsRule.ts#L22L51))